ui/app/src/Main.elm

```diff
@@ -86,12 +86,24 @@ init flags url key =
 
             else
                 "/"
+
+        startWeekAtMonday =
+            flags
+                |> Json.decodeValue (Json.field "startWeekAtMonday" Json.bool)
+                |> Result.withDefault False
```

Comment:
Perhaps we should reverse the logic and make Monday the default if we fail to decode.